etl/steps/data/garden/who/2024-07-26/mortality_database.py

```diff
@@ -17,7 +14,7 @@ def run(dest_dir: str) -> None:
     # Load inputs.
     #
     # Load meadow dataset.
-    ds_meadow = cast(Dataset, paths.load_dependency("mortality_database"))
+    ds_meadow = paths.load_dataset("mortality_database")
 
     # Read table from meadow dataset.
     tb = ds_meadow["mortality_database"]
```

Comment:
I have learned from Pablo that you can now combine getting the table and reset index by using tb = ds_meadow.read_table("mortality_database") :)

Reply:
Ah no way, I didn't know that, thanks!